core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -733,6 +733,15 @@ private static int getSlaveAgentPortInitialValue(int def) {
      */
     private transient final AdjunctManager adjuncts;
 
+    /**
+     * Interval in seconds between 2 ajax calls.
+     */
+    private int ajaxRefreshInterval = 6;
```

Comment:
I don't know how to configure it right now... still in progress

Reply:
[`SystemProperties`](https://javadoc.jenkins.io/jenkins/util/SystemProperties.html) ?

Reply:
it was to be sure that it's working :), I will change it to 5 before moving out of draft.

Reply:
done.